modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue

```diff
@@ -524,6 +525,14 @@ export default {
       this.refresh();
     });
 
+    // sessionStorage because it is deliberately tab-specific
```

Comment:
Can we specify "browser tab" since we have tabs in our UI? I was confused for a minute there.